### PR TITLE
Message retries failing and Endpointconfiguration not updating

### PR DESCRIPTION
**Pull request by petersgiles**
#202 #203

### CODE REVIEW

**Comment by petersgiles** [comment]
@johnsimons do you want to review?

**Comment by johnsimons** [comment]
The change looks good but I saw lots of notification commented out:
- https://github.com/Particular/ServicePulse/pull/206/files#diff-2e8e4aea7512a76953509cb663ed9ce2L136
- https://github.com/Particular/ServicePulse/pull/206/files#diff-2e8e4aea7512a76953509cb663ed9ce2R194

Are we not providing any feedback to the user ?

**Comment by petersgiles** [comment]
@johnsimons those notifications have been replaced by all those animations. I need to clean those commented items up.

File: src/ServicePulse.Host/app/js/configuration/configuration.controller.js [hunk 1]
```diff
@@ -1,26 +1,71 @@
...
-            serviceControlService.updateEndpoint(id, { "monitor_heartbeat": newState });
+         
+            configurationService.update(id, monitor, 'Updating', 'Update Failed')
+                .then(function (message) {
+                    result.workflow_state = createWorkflowState('ready', message);
+                    result.monitor_heartbeat = monitor;
+                }, function (message) {
+                    result.workflow_state = createWorkflowState('error', message);
+                })
+                .finally(function () {
```
Comment (johnsimons) [inline src/ServicePulse.Host/app/js/configuration/configuration.controller.js]: empty?

**Comment by johnsimons** [comment]
@petersgiles I don't see anything odd with it but maybe @colin-higgins could give it another set of eyes

**Pull request merged**